src/AutoRest.CSharp/Common/Generation/Writers/FormattableStringHelpers.cs

```diff
@@ -106,7 +106,21 @@ public static FormattableString GetConversionFormattable(this Parameter paramete
                     return $"{typeof(BinaryData)}.{nameof(BinaryData.FromObjectAsJson)}({(enumType.IsIntValueType ? $"({enumType.ValueType}){parameter.Name}" : $"{parameter.Name}.{enumType.SerializationMethodName}()")})";
                 }
             }
+            /*
+            if (parameter.Type is { IsFrameworkType: true } && parameter.Type.EqualsIgnoreNullable(typeof(string)) && toType.EqualsIgnoreNullable(typeof(RequestContent)))
+            {
+                return $"{typeof(RequestContentHelper)}.{nameof(RequestContentHelper.FromString)}({parameter.Name})";
+            }
 
+            if (parameter.Type is { IsFrameworkType: true } && parameter.Type.EqualsIgnoreNullable(typeof(bool)) && toType.EqualsIgnoreNullable(typeof(RequestContent)))
+            {
+                return $"{typeof(RequestContentHelper)}.{nameof(RequestContentHelper.FromBool)}({parameter.Name})";
+            }
+            */
+            if (parameter.Type is { IsFrameworkType: true } && toType.EqualsIgnoreNullable(typeof(RequestContent)))
+            {
+                return $"{typeof(RequestContentHelper)}.{nameof(RequestContentHelper.FromObject)}({parameter.Name})";
+            }
```

Comment:
shall we have a refactor of these since they all have the same condition: `toType is RequestContent`?

Reply:
updated.